<launch>
  <arg name="velocity_gain" default="1.5"/>
  <arg name="angular_gain" default="1.5"/>
  <arg name="maximum_limit_vel" default="0.5"/>
  <arg name="minimum_limit_vel" default="-0.5"/>
  <arg name="in_twist" default="raw_cmd_vel"/>
  <arg name="out_twist" default="filtered_cmd_vel"/>

  <node pkg="velocity_smoother" exec="velocity_smoother_node" name="velocity_smoother_node">
    <param name="velocity_gain" value="$(var velocity_gain)"/>
    <param name="angular_gain" value="$(var angular_gain)"/>
    <param name="maximum_limit_vel" value="$(var maximum_limit_vel)"/>
    <param name="minimum_limit_vel" value="$(var minimum_limit_vel)"/>
    <remap from="raw_cmd_vel" to="$(var in_twist)"/>
    <remap from="filtered_cmd_vel" to="$(var out_twist)"/>
  </node>
</launch>
